benchmarks/src/generated/main/grpc/io/grpc/benchmarks/proto/BenchmarkServiceGrpc.java

```diff
@@ -38,6 +38,7 @@ private BenchmarkServiceGrpc() {}
               io.grpc.benchmarks.proto.Messages.SimpleRequest.getDefaultInstance()))
           .setResponseMarshaller(io.grpc.protobuf.ProtoUtils.marshaller(
               io.grpc.benchmarks.proto.Messages.SimpleResponse.getDefaultInstance()))
+          .setSchemaDescriptor(new BenchmarkServiceFileDescriptorSupplier().getServiceDescriptor().findMethodByName("UnaryCall"))
```

Comment:
This should have just been `.setSchemaDescriptor(new BenchmarkServiceFileDescriptorSupplier())`